core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -130,10 +130,41 @@ public ContractMethod(Caver caver, String type, String name, List<ContractIOType
         return ABI.decodeParameters(matchedMethod, encodedResult);
     }
 
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * When creating CallObject, it need not to fill 'data', 'to' fields.
+     * The 'data', 'to' fields automatically filled in call() method.
+     * It is recommended to use this function when you want to execute one of the functions with the same number of parameters.
+     * @param arguments A List of parameter that solidity wrapper type to call smart contract method.
+     * @param callObject A CallObject instance to 'call' smart contract method.
+     * @return List
+     * @throws IOException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> callWithSolidityWrapper(List<Type> arguments, CallObject callObject) throws IOException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+        List<Type> functionParams = new ArrayList<>();
+
+        if(arguments != null) {
+            functionParams.addAll(arguments);
+        }
+
+        String encodedFunction = this.encodeABIWithSolidityWrapper(arguments);
+
+        if(callObject.getData() != null || callObject.getTo() != null) {
+            LOGGER.warn("'to' and 'data' field in CallObject will overwrite.");
+        }
+        callObject.setData(encodedFunction);
+        callObject.setTo(this.getContractAddress());
+        Bytes response = caver.rpc.klay.call(callObject).send();
+
+        String encodedResult = response.getResult();
+        return ABI.decodeParameters(this, encodedResult);
```

Comment:
How about implementing this duplicated part to implement private function separately and call it internally?